src/components/WebexMeeting/WebexMeeting.js

```diff
@@ -0,0 +1,46 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {Spinner} from '@momentum-ui/react';
+
+import {useMeeting} from '../hooks';
+
+import {WebexInMeeting, WebexInterstitialMeeting, WebexMeetingControl, WebexMeetingControls} from '..';
+
+import './WebexMeeting.css';
+
+/**
+ * Webex Meeting component displays all meeting contents.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexMeeting({meetingDestination, setControls}) {
+  const {ID, remoteVideo} = useMeeting(null, meetingDestination);
+
+  const isActive = remoteVideo !== null;
+
+  const controls = setControls(isActive).map((key) => <WebexMeetingControl type={key} key={key} />);
+  const meeting = ID ? (
+    <Fragment>
+      {isActive ? <WebexInMeeting meetingID={ID} /> : <WebexInterstitialMeeting meetingID={ID} />}
+      <div className="controls">
```

Comment:
Does `WebexMeetingControls` create a `div` itself? if not, I think this should go within that component, since `WebexMeetingControls` will always wrap `WebexMeetingControl`